<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>147</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MySNS</string>
  </property>
  <widget class="QLineEdit" name="idLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your pw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>89</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="signUpCheckbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>93</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string> Sign up</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="signOutCheckbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>93</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string> Sign out</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="findCheckbox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>93</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string> Find id/pw</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
